remote.go

```diff
@@ -1148,6 +1148,46 @@ func (wd *remoteWD) execScript(script string, args []interface{}, suffix string)
 	return reply.Value, nil
 }
 
+func (wd *remoteWD) execCdpCommandRaw(data []byte) ([]byte, error) {
+	return wd.execute("POST", wd.requestURL("/session/%s/goog/cdp/execute", wd.id), data)
+}
+
+// execCdpCommand execute cdp command
+// this interface didn't define in official wiki(https://github.com/SeleniumHQ/selenium/wiki/JsonWireProtocol)
+// so I just refered to selenium in python (selenium.webdriver.chrome.remote_connection.ChromeRemoteConnection)
```

Comment:
Since the documentation is provided in the interface, we don't need to repeat it here.

Reformat this comment as:

// This command is not defined in the Selenium or WebDriver documentation. The functionality was ported from the Python Selenium driver (selenium.webdriver.chrome.remote_connection.ChromeRemoteConnection).

(Add line breaks at 80ish characters; it is hard for me to do this accurately in this tiny review comment text box).